### ISSUE TITLE
Got invalid selector error with pager 0.1.0 as dependency

### ISSUE THREAD

**Issue by chemzqm** [retrofox/datatable#4]
The property `el` of  `component/pager` is a List instance, the jQuery instance is not acceptable with it's method `appendTo` .
I could add `[0]` to make it works in line 187:

```
pager.el.appendTo(this.el.find('tfoot td')[0]);
```

**Comment by retrofox** [comment]
Yes @chemzqm , you're right. We could use .get() jQuery method instead of []. Something like 

``` js
pager.el.appendTo(this.el.find('tfoot td').get(0));
```

Let me fix it.

**Comment by retrofox** [comment]
I've fixed it in the new `0.1.2` release. Thxs @chemzqm.
